<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-UNQEGMQ7" failures="5" tests="7" name="用户管理系统测试用例" time="9.994" errors="0" timestamp="04 7月 2022 00:43:18 GMT">
  <testcase classname="com.course.cases.LoginTest" name="loginTrue" time="3.715"/>
  <testcase classname="com.course.cases.LoginTest" name="loginFalse" time="3.066"/>
  <testcase classname="com.course.cases.AddUserTest" name="addUser" time="0.027">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.course.cases.AddUserTest.addUser(AddUserTest.java:29)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- addUser -->
  <testcase classname="com.course.cases.GetUserInfoTest" name="getUserInfo" time="3.056">
    <failure type="java.lang.AssertionError" message="expected [[{&amp;quot;password&amp;quot;:&amp;quot;123456&amp;quot;,&amp;quot;isDelete&amp;quot;:null,&amp;quot;sex&amp;quot;:null,&amp;quot;permission&amp;quot;:null,&amp;quot;id&amp;quot;:1,&amp;quot;userName&amp;quot;:&amp;quot;zhangsan&amp;quot;,&amp;quot;age&amp;quot;:null}]] but found [[{&amp;quot;password&amp;quot;:&amp;quot;123456&amp;quot;,&amp;quot;id&amp;quot;:1,&amp;quot;userName&amp;quot;:&amp;quot;zhangsan&amp;quot;}]]">
      <![CDATA[java.lang.AssertionError: expected [[{"password":"123456","isDelete":null,"sex":null,"permission":null,"id":1,"userName":"zhangsan","age":null}]] but found [[{"password":"123456","id":1,"userName":"zhangsan"}]]
	at com.course.cases.GetUserInfoTest.getUserInfo(GetUserInfoTest.java:42)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- getUserInfo -->
  <testcase classname="com.course.cases.UpdateUserInfoTest" name="updateUserInfo" time="0.028">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.course.cases.UpdateUserInfoTest.updateUserInfo(UpdateUserInfoTest.java:25)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- updateUserInfo -->
  <testcase classname="com.course.cases.UpdateUserInfoTest" name="deleteUser" time="0.027">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.course.cases.UpdateUserInfoTest.deleteUser(UpdateUserInfoTest.java:63)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- deleteUser -->
  <testcase classname="com.course.cases.GetUserInfoListTest" name="getUserListInfo" time="0.025">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at com.course.cases.GetUserInfoListTest.getUserListInfo(GetUserInfoListTest.java:27)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- getUserListInfo -->
</testsuite> <!-- 用户管理系统测试用例 -->
